packages/amplify-e2e-tests/src/configure_tests.ts

```diff
@@ -0,0 +1,39 @@
+import { config } from 'dotenv';
+import { existsSync } from 'fs';
+import { homedir } from 'os';
+import { join } from 'path';
+
+import configure from './configure';
+import { isCI } from './utils';
+
+const AWS_CREDENTIAL_PATH = join(homedir(), '.aws', 'credentials');
+
+async function setUpAmplify() {
+  if (isCI()) {
+    let AWS_ACCESS_KEY_ID;
+    let AWS_SECRET_ACCESS_KEY;
+    AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID;
+    AWS_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY;
+    if (!AWS_ACCESS_KEY_ID || !AWS_SECRET_ACCESS_KEY) {
+      throw new Error(
+        'Please set AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY in .env or as Environment variable in CircleCI'
+      );
+    }
+    await configure({
+      accessKeyId: AWS_ACCESS_KEY_ID,
+      secretAccessKey: AWS_SECRET_ACCESS_KEY,
+      profileName: 'amplify-integ-test-user'
+    });
+  } else {
```

Comment:
Not sure why this is an else for isCI()

Reply:
It was for debugging, I can get rid of it